// sanity-plugin-podcast/lib/podcast.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;

var _fa = require("react-icons/fa");

var _languages = _interopRequireDefault(require("./languages"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var _default = {
  name: 'podcast',
  title: 'Podcast',
  description: 'Create and configure a podcast',
  icon: _fa.FaPodcast,
  type: 'document',
  fields: [{
    name: 'title',
    type: 'string',
    required: true,
    description: 'Remember that if your title is too long, it may be truncated in various podcatchers-'
  }, {
    name: 'subtitle',
    type: 'string',
    description: 'That catchy tagline.'
  }, {
    name: 'slug',
    title: 'Podcast slug',
    type: 'slug',
    description: 'For when you need to refer to your podcast in a url.',
    options: {
      source: 'title',
      slugify: input => input.toLowerCase().replace(/\s+/g, '-').slice(0, 200)
    }
  }, {
    name: 'description',
    type: 'text',
    required: true,
    description: 'What is this podcast about and why should people subscribe to it?'
  }, {
    name: 'coverArt',
    title: 'Cover art',
    type: 'image',
    required: true,
    description: 'The image should be either jpg or png. Preferably 3000 x 3000, minimum 1400 x 1400 pixels.'
  }, {
    name: 'copyright',
    type: 'string',
    description: 'Who owns the rights to this podcast?'
  }, {
    name: 'language',
    type: 'string',
    description: 'What language is this podcast in?',
    options: {
      list: _languages.default
    }
  }, {
    name: 'explicit',
    type: 'boolean',
    description: 'Do you need to warn parents about the content in this podcast? (You can set this for individual episodes to)'
  }, {
    name: 'itunes',
    title: 'iTunes settings',
    type: 'itunes',
    description: 'iTunes has extra stuff'
  }],
  preview: {
    select: {
      title: 'title',
      subtitle: 'subtitle',
      description: 'description',
      media: 'coverArt'
    }
  }
};
exports.default = _default;
